lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -496,12 +513,12 @@ void amf_app_state_free_ue_context(void** ue_context_node) {
  ***************************************************************************/
 tmsi_t amf_lookup_guti_by_ueid(amf_ue_ngap_id_t ue_id) {
   amf_context_t* amf_ctxt = amf_context_get(ue_id);
-
+  OAILOG_FUNC_IN(LOG_AMF_APP);
   if (amf_ctxt == NULL) {
-    return (0);
+    OAILOG_FUNC_RETURN(LOG_AMF_APP, 0);
```

Comment:
I think we need to return "INVALID_TMSI"

Reply:
The caller of this function expects "0" if guti is not found.
The value of "INVALID_TMSI" is UINT32_MAX.